node/resolvers/index.ts

```diff
@@ -9,21 +22,69 @@ export const resolvers = {
       const result = await ctx.clients.licenseManagerClient.getProducts()
       return result.items
     },
-    product: (_: any, { id }: productParams, ctx: AppContext): Product =>
+    product: (_: any, { id }: ProductParams, ctx: AppContext): Product =>
       ctx.clients.licenseManagerClient.getProduct(id),
+    resource: async (
+      _: any,
+      { productId, resourceKey }: ResourceParams,
+      ctx: AppContext
+    ): Promise<Resource> => {
+      const resource = await ctx.clients.licenseManagerClient.getResource(
+        productId,
+        resourceKey
+      )
+
+      return {
+        id: resource.Id,
+        cacheId: resource.Id,
+        name: resource.Name,
+        resourceKey: resource.ResourceKey,
+        categoryId: resource.CategoryId,
+        description: resource.Description,
+        statements: resource.Statements,
+        licenses: resource.Licenses
+      }
+    },
+    resourceCategories: (
+      _: any,
+      { productId }: ResourceCategoriesParams,
+      ctx: AppContext
+    ): ResourceCategory[] => {
+      return ctx.clients.licenseManagerClient.getResourceCategories(productId)
+    },
+    licenses: async (_: any, __: any, ctx: AppContext): Promise<License[]> => {
+      const result = await ctx.clients.licenseManagerClient.getLicenses()
+      return result.items
+    }
   },
   Product: {
-    id: (lmProduct: lmProduct, _: any, __: any): ID => `${lmProduct.id}`,
-    cacheId: (lmProduct: lmProduct, _: any, __: any): ID => `${lmProduct.id}`,
-    resources: (lmProduct: lmProduct, _: any, __: any): lmResource[] =>
+    id: (lmProduct: LmProduct, _: any, __: any): ID => `${lmProduct.id}`,
```

Comment:
We only use `_: any` if they are keeping the place of a param that we're not using that is before a param that we are using, for example at line 55. If they are after all params that you're using, you don't actually need them.
```suggestion
    id: (lmProduct: LmProduct): ID => `${lmProduct.id}`,
```